src/current/_includes/common/telemetry-table.html

```diff
@@ -0,0 +1,932 @@
+      "dependencies": null
+    }
+</pre>
+  </td>
+ </tr>
+ <tr>
+  <td>CPU information</td>
+  <td>Information about the CPU on the machine hosting CockroachDB.</td>
+  <td>
+   <ul>
+    <li>Number of CPUs</li>
+    <li>Sockets</li>
+    <li>Cores</li>
+    <li>CPU Model</li>
+    <li>CPU Speed</li>
+    <li>Features</li>
+    <ul>
+     <li>CPU feature flags for first CPU</li>
+    </ul>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"cpu": {
+        "numcpu": 16,
+        "sockets": 16,
+        "cores": 1,
+        "model": "Intel(R) Xeon(R) CPU @ 2.60GHz",
+        "mhz": 2600.032,
+        "features": [
+            "fpu",
+            "tsc",
+            "pse",
+            "sse4_2",
+            "aes",
+            "avx",
+            "avx2",
+            "rdseed",
+            "smap",
+            "hypervisor"
+        ]
+    }  
+</pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Memory information</td>
+  <td>Information about the memory on the machine hosting CockroachDB.</td>
+  <td>
+   <ul>
+    <li>Total Memory</li>
+    <li>Available Memory</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"mem": {
+        "total": "67413340264",
+        "available": "38576678952"
+      }
+  </pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Hardware information</td>
+  <td>Information about the hardware on the machine hosting CockroachDB.</td>
+  <td>
+   <ul>
+    <li>Virtualization Type</li>
+    <li>CPU Info</li>
+    <li>Memory Info</li>
+    <li>Load Average</li>
+    <li>Cloud Provider</li>
+    <li>Cloud Provider Instance Class</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"virtualization": ""
+"loadavg15": 7.10,
+"provider": "gcp",
+"instanceClass": "n2-standard-16"
+  </pre>
+See also CPU Info and Memory Info
+</td>
+ </tr>
+ <tr>
+  <td>OS information</td>
+  <td>Information about the operating system on the machine hosting CockroachDB.</td>
+  <td>
+   <ul>
+    <li>OS Family</li>
+    <li>Platform</li>
+    <li>Version</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+ "os": {
+      "family": "debian",
+      "platform": "ubuntu",
+      "version": "22.04"
+    }
+  </pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Topology information</td>
+  <td>Information about the topology and/or cloud infrastructure hosting CockroachDB.</td>
+  <td>
+   <ul>
+    <li>Cloud Provider</li>
+    <li>Region</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"topology": {
+      "provider": "gcp",
+      "region": "us-east1-b"
+    }
+</pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Environment</td>
+  <td>Details of the environment where the CockroachDB node or SQL instance is running.</td>
+  <td>
+   <ul>
+    <li>Locality</li>
+    <li>Hardware Info</li>
+    <li>OS Info</li>
+    <li>Build Info</li>
+    <li>License Type</li>
+    <li>Topology</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"licenseType": "Enterprise"
+  </pre>
+See also: Locality, Hardware Info, OS Info, Build Info, and Topology Info
+</td>
+ </tr>
+ <tr>
+  <td>License</td>
+  <td>Details about the cluster’s license.</td>
```

Comment:
Replace smart with straight quotes

```suggestion
  <td>Details about the cluster's license.</td>
```

Reply:
✅